libs/onboarding-sdk-crypto/pom.xml

```diff
@@ -75,6 +75,25 @@
             <version>2.1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.http-client</groupId>
+            <artifactId>google-http-client</artifactId>
+            <version>1.40.1</version> <!-- Check for the latest version -->
```

Comment:
you could use a property <google-http-sdk-version> for version in order to use also in the below dependencies